src/javascript/app/base/socket_general.js

```diff
@@ -37,6 +38,8 @@ const BinarySocketGeneral = (() => {
                     is_available = /^up$/i.test(response.website_status.site_status);
                     if (is_available && !BinarySocket.availability()) {
                         window.location.reload();
+                    } else if (is_available && response.website_status.message) {
+                        Footer.displayNotification(response.website_status.message);
                     } else if (!is_available) {
                         Header.displayNotification(response.website_status.message, true);
                     }
```

Comment:
we could change the block, so it becomes:
```
if (is_available) {
    if (!BinarySocket.availability()) {
        window.location.reload();
    } else if (response.website_status.message)
        Footer.displayNotification(response.website_status.message);
    }
} else {
    Header.displayNotification(response.website_status.message, true);
}
```
but it's up to you, this is fine as well